mautic/mautic
# Added option to easily embed form via iframe

Added option to easily embed a form via iframe. It could be handy for systems which doesn't support additional `<script/>` tags or if there is a JS/CSS conflict or whatever other reason the JS embed won't work.

![mautic-form-iframe](https://cloud.githubusercontent.com/assets/1235442/10727019/f8bf2a20-7bd6-11e5-9bd1-2c14589a3930.png)

### Testing
1. Open detail of a form.
2. Click on Automatic embed.
3. Copy the iframe code.
4. Paste the iframe code to a HTML page.

Result: The form should appear in the page.
